exonum/src/blockchain/schema.rs

```diff
@@ -263,10 +277,14 @@ where
         }
         panic!(
             "Couldn't not find any config for height {}, \
-                that means that genesis block was created incorrectly.",
+             that means that genesis block was created incorrectly.",
             height
         )
     }
+
+    fn next_height(&self) -> Height {
```

Comment:
This function isn't public, but I propose to add a doc-comment too. 
There is a possibility to generate documentation for a "private" api by the way.